README.md

```diff
@@ -17,23 +23,42 @@ way most fit for purpose, and a return call can be made to the server to unblock
 the read in progress and so direct sequencing capacity towards reads of interest.
 
 
-**This implementation of ReadFish requires Guppy version >= 6.0.0 and MinKNOW version core >= 5.0.0 . It will not work on earlier versions.**
+**This implementation of readfish requires Guppy version >= 6.0.0 and MinKNOW version core >= 5.0.0 . It will not work on earlier versions.**
 
 
-**Currently we only recommend LINUX for running ReadFish. We have not had
-effective performance on other platforms to date.**
-
 The code here has been tested with Guppy in GPU mode using GridION Mk1 and
-NVIDIA RTX2080 on live sequencing runs and an NIVIDA GTX1080 using playback
+NVIDIA RTX2080 on live sequencing runs and an NVIDIA GTX1080 using playback
 on a simulated run (see below for how to test this).
 This code is run at your own risk as it DOES affect sequencing output. You
 are **strongly** advised to test your setup prior to running (see below for
 example tests).
 
+## Supported Sequencing Platforms
+
+The following platforms are supported:
+
+- **PromethION** Big Boy
+- **GridION** Box
+- **MinION** Smol Boy
+
```

Comment:
Please can we add in P2solo?
It could be the Smol But Powerful Boi?
Or - alternatively - we could just give the names and drop the YOOF speak.